src/sh/sdv-kanto-ctl

```diff
@@ -0,0 +1,422 @@
+	
+	jq \
+	--arg REGISTRY_HOSTNAME "$REGISTRY_HOSTNAME" \
+	'del(.containers.registry_configurations)' < ${KANTO_CONFIG} > ${TMPFILE}
+	
+	[ "$VERBOSE" = "1" ] && echo "Created temp file for modifications: ${TMPFILE}"
+
+	if [ -w ${KANTO_CONFIG} ]; then
+		if ! diff ${TMPFILE} ${KANTO_CONFIG} > /dev/null; then
+			BACKUP=$(mktemp --tmpdir kanto-cm-config-backup.XXXXXXX)
+			echo "Backing up to ${BACKUP}"
+			echo "Applying changes:"
+			cp ${KANTO_CONFIG} ${BACKUP}
+			cp ${TMPFILE} ${KANTO_CONFIG}
+			jq < ${KANTO_CONFIG} 
+			[ "$VERBOSE" = "1" ] &&	echo "Ok, changes applied."
+			CHANGES_APPLIED=1
+		else
+			echo "No difference to existing configuration."
+		fi
+	else
+		echo "Unable to apply changes:"
+		jq < ${TMPFILE} 
+		echo "ERROR: No write permissions to ${KANTO_CONFIG}"
+		echo "The changes cannot be applied automatically, as the current user does not have write permissions to ${KANTO_CONFIG}"
+		echo "Either apply the configuration changes manually, or run this script with write permissions (e.g. use sudo):"
+		echo "Run: sudo cp ${TMPFILE} ${KANTO_CONFIG}"
+		exit 1
+	fi
+}
+
+list_registries() {
+	jq '.containers.registry_configurations | keys[]' < ${KANTO_CONFIG}
+}
+
+show_config() {
+	jq < ${KANTO_CONFIG}
+}
+
+set_config() {
+	echo "Setting configuration $CONFIG_KEY to $CONFIG_VALUE"
+	TMPFILE=$(mktemp --tmpdir kanto-cm-config.XXXXXXX)
+	CONFIG_KEY=".${CONFIG_KEY}"
+	jq \
+	--arg CONFIG_KEY "$CONFIG_KEY" \
+	--arg CONFIG_VALUE "$CONFIG_VALUE" \
+	'setpath( $CONFIG_KEY|split(".")[1:]; $CONFIG_VALUE)' < ${KANTO_CONFIG} > ${TMPFILE}
+	[ "$VERBOSE" = "1" ] && echo "Created temp file for modifications: ${TMPFILE}"
+
+	if [ -w ${KANTO_CONFIG} ]; then
+		if ! diff ${TMPFILE} ${KANTO_CONFIG} > /dev/null; then
+			BACKUP=$(mktemp --tmpdir kanto-cm-config-backup.XXXXXXX)
+			echo "Backing up to ${BACKUP}"
+			cp ${KANTO_CONFIG} ${BACKUP}
+			cp ${TMPFILE} ${KANTO_CONFIG}
+			[ "$VERBOSE" = "1" ] &&	echo "Ok, changes applied."
+			CHANGES_APPLIED=1
+		else
+			echo "No difference to existing configuration."
+		fi
+	else
+		echo "ERROR: No write permissions to ${KANTO_CONFIG}"
+		jq < ${TMPFILE}
+		echo "The changes cannot be applied automatically, as the current user does not have write permissions to ${KANTO_CONFIG}"
+		echo "Either apply the configuration changes manually, or run this script with write permissions (e.g. use sudo):"
+		echo "Run: sudo cp ${TMPFILE} ${KANTO_CONFIG}"
+		exit 1
+	fi
+
+}
+
+######################################################
+#                        main                        #
+######################################################
+
+if [ -z "$COMMAND" ]; then
+	display_help
+	exit 0
+fi 
+
+[ "$COMMAND" = "add-registry" ] && add_registry
+[ "$COMMAND" = "remove-registry" ] && remove_registry
+[ "$COMMAND" = "remove-all-registries" ] && remove_all_registries
+[ "$COMMAND" = "list-registries" ] && list_registries
+[ "$COMMAND" = "show-config" ] && show_config
+[ "$COMMAND" = "set" ] && set_config
```

Comment:
Make commands more consistent, e.g.
"show-config" + "set-config"
vs.
"show" + "set"

Reply:
Renamed command to "set-value" to make it more consistent